neptune/new/model_version.py

```diff
@@ -65,7 +67,19 @@ def __init__(
     def _label(self) -> str:
         return self._sys_id
 
-    def change_stage(self, stage: str, wait=False):
-        self._op_processor.enqueue_operation(
-            ChangeStage(container_id=self._id, stage=ModelVersionStage(stage)), wait
+    def change_stage(self, stage: str):
+        stage = ModelVersionStage(stage)
+
+        if self._op_processor.mode == Mode.OFFLINE:
+            raise NeptuneOfflineModeChangeStageException()
```

Comment:
I wasn't 100% sure about adding `OperationProcessor.model` attribute, but I think it's better than checking `op_processor` class type.

Reply:
I guessss. Thing is `SyncOperationProcessor` is used in `mode == Mode.DEBUG`, which cause a little bit o weird situation for me

Reply:
Right, I didn't notice that. Let's don't use mode then.